src/components/Carousel.jsx

```diff
@@ -27,11 +27,11 @@ const Carousel = ({
           const { id } = item;
 
           if (item?.name) {
-            return <Videogame key={id} data={item} />;
+            return <Videogame key={id} data={item} deleteButton={deleteButton} />;
           }
 
           return (
-            <Reservation key={id} data={item} />
+            <Reservation key={id} data={item} deleteButton={deleteButton} />
```

Comment:
Would it make sense not to add this prop to `Reservation`? Currently, the component is not expecting or implementing it.